go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	github.com/soheilhy/cmux v0.1.4 // indirect
 	github.com/spf13/afero v1.2.2 // indirect
 	github.com/spf13/cobra v0.0.3 // indirect
+	github.com/spf13/pflag v1.0.1
```

Comment:
Rebased on top of https://github.com/digitalocean/digitalocean-cloud-controller-manager/pull/229